src/blockdata/transaction.rs

```diff
@@ -462,15 +462,22 @@ impl Transaction {
         }
     }
 
+    #[cfg(feature="bitcoinconsensus")]
+    /// Shorthand for [Self::verify_with_flags] with flag [bitcoinconsensus::VERIFY_ALL]
+    pub fn verify<S>(&self, spent: S) -> Result<(), script::Error>
+        where S: FnMut(&OutPoint) -> Option<TxOut> {
+        self.verify_with_flags(spent, ::bitcoinconsensus::VERIFY_ALL)
+    }
+
     #[cfg(feature="bitcoinconsensus")]
     /// Verify that this transaction is able to spend its inputs
     /// The lambda spent should not return the same TxOut twice!
-    pub fn verify<S>(&self, mut spent: S) -> Result<(), script::Error>
-        where S: FnMut(&OutPoint) -> Option<TxOut> {
+    pub fn verify_with_flags<S, F>(&self, mut spent: S, flags: F) -> Result<(), script::Error>
+        where S: FnMut(&OutPoint) -> Option<TxOut>, F : Into<u32> + Copy {
```

Comment:
Not sure why we need to require flags to be `Copy` here

Reply:
The reason was that flags is used in a for, but yeah I solved by creating the u32 before in d1f4c0a5c863d4e12f5a49fed4798a4a7764583c